app/livechat/client/views/app/livechatAgents.js

```diff
@@ -2,15 +2,20 @@ import { Meteor } from 'meteor/meteor';
 import { Template } from 'meteor/templating';
 import { ReactiveVar } from 'meteor/reactive-var';
 import { ReactiveDict } from 'meteor/reactive-dict';
-import s from 'underscore.string';
 import _ from 'underscore';
 
 import { modal, call } from '../../../../ui-utils';
 import { t, handleError, APIClient } from '../../../../utils/client';
 import './livechatAgents.html';
 
-const loadAgents = async (instance, limit = 50) => {
-	const { users } = await APIClient.v1.get(`livechat/users/agent?count=${ limit }`);
+const loadAgents = async (instance, limit = 50, text) => {
+	let baseUrl = `livechat/users/agent?count=${ limit }`;
+
+	if (text) {
+		baseUrl = baseUrl.concat(`&text=${ encodeURIComponent(text) }`);
```

Comment:
Please use assign operators to concat (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/concat#Performance)
```suggestion
		baseUrl += `&text=${ encodeURIComponent(text) }`;
```